docs/authentication/classic-token.md

```diff
@@ -0,0 +1,15 @@
+# "Classic" Personal Access Token (PAT) Requirements and Risks
+Certain features require a Personal Access Token (PAT).
+We recommend you use a fine-grained token as described in [Authentication](/docs/authentication/fine-grained-auth-token.md).
```

Comment:
```suggestion
We recommend you use a fine-grained token as described in [Authentication with Fine-grained PAT](/docs/authentication/fine-grained-auth-token.md).
```

or "as described [here]" to avoid repetition.

"in [Authentication]" seems odd given that this file is also in docs/authentication.